docs/components/connectors/out-of-the-box-connectors/slack.md

```diff
@@ -35,6 +35,11 @@ Set **Authentication** to your Slack OAuth token, which is stored as a secret. F
 
 ### Create channel
 
+:::info
+This API uses the Slack [conversations.create](https://api.slack.com/methods/conversations.create) method.
```

Comment:
Please wrap conversations.create in ``

`conversations.create`

Reply:
Done